homeassistant/components/tado/__init__.py

```diff
@@ -268,6 +268,7 @@ def set_presence(
             self.tado.setAway()
         elif presence == PRESET_HOME:
             self.tado.setHome()
+        self.update_zones()
```

Comment:
I guess this works for now. 

The Tado integration would, however, greatly benefit from a DataUpdateCoordinator to handle all these sorts of things. Maybe consider implementing such a thing for a followup PR?

../Frenck